docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -75,12 +76,15 @@ function uiowa_core_theme($existing, $type, $theme, $path) {
 function template_preprocess_uiowa_core_headline(&$variables) {
   if ($variables['hide_headline'] == TRUE) {
     $style = 'sr-only';
+    $alignment = 'sr-only';
```

Comment:
Though this might solve the error, I am not sure this is how we should go about it. Going to poke at this a bit

Reply:
the `if` check only related to hiding the headline which doesn't change what we do with alignment. moved it in https://github.com/uiowa/uiowa/pull/4834/commits/4006eba21d74017ca75bae6c58bbc5349fd131e1

Reply:
AWESOME, thank you Joe!!